### DISCUSSION TITLE
gagal scan QR di Beta WA ( beta version )

### DISCUSSION THREAD

**Discussion by bmd-rizqifauzan** [kirimwa-id/forum#34] [Tanya Jawab] [answered]
saya menggunakan linked device dimana saya bisa menggunakan wa-web tanpa HP

saat saya scan qr-code dengan wa-saya, tidak terjadi apa apa
dan wa saya tidak terhubung

**Comment by rioastamal** [comment, marked as answer]
API KirimWA.id saat ini belum mendukung WhatsApp Multi-Device Beta. Solusinya adalah menggunakan WhatsApp versi stabil. Dukungan untuk WhatsApp Multi-Device Beta akan segera dimasukkan ke Roadmap.

**Comment by rioastamal** [comment]
Roadmap untuk WhatsApp Multi-Device Beta dapat diikuti pada thread ini #42
